src/components/FilteredRecipe/FilteredRecipe.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { filter } from "./filter";
+
+const FilteredRecipe = ({ selectedFilter, handleFilter }) => {
+  return (
+    <div className="my-24 px-20 text-center">
+      <h1 className="text-4xl mb-2">Recipes</h1>
+      <h4 className="cooking font-bold mb-6">GET COOKING & SHARING</h4>
+      <div className="flex flex-wrap justify-center items-center">
+        <p className="mr-4">FILTER BY:</p>
+        {filter.map((item, index) => (
+          <button
+            key={index}
```

Comment:
Can you remember what I said about using `index` as key? 🤔 Perhaps better to use something that's a bit more unique.